### ISSUE TITLE
update self.date in cnn-lstm notebook

### ISSUE THREAD

**Issue by Seiris21** [Seiris21/2022_snowpack_capstone#1]
This was an unpushed change when AWS went down. 

Line: 
`def get_default_transforms():
    transform = {
        "train": T.Compose(
            [
                #T.RandomHorizontalFlip(),
                #T.RandomVerticalFlip(),
                #T.RandomAffine(15, translate=(0.1, 0.1), scale=(0.9, 1.1)),
                T.ColorJitter(brightness=0.3, contrast=0.3, saturation=0.3),
                T.ConvertImageDtype(torch.float),
                T.Normalize(mean = (0.485, 0.456, 0.406), 
                            std = (0.229, 0.224, 0.225))
                `

`        #Pull from weather data and generate time-series
        date_range = pd.date_range(end=self.date[index], periods=self.seq_len)
        ts = []
        for date in date_range:
            query = self.timeseries.loc[(self.timeseries['cell_id']==self.cell_id[index]) & (self.timeseries['date']==date)] <<<<-----------------------
            if not query.empty:
                ts.append([list(query['HRRR_TMP_surface_12h'])[0],list(query['HRRR_PRATE_surface_12h'])[0]])
            else:
                ts.append([np.nan,np.nan])
        ts = torch.tensor(ts)`
